### PR TITLE
French localization  part 2

### PR DESCRIPTION
**Pull request by ValentinChCloud**
PR #5089 have been accepted, we are calling people to translate approximately 190 new sentences for the french dictionnary. 
As said in #3762 , S'il vous plaît aider si vous parlez français :)

Stop doing a hard job and come and enjoy, time of a Pull Request 🎈 

@yvanlebras @abretaud @lecorguille @bebatut @mvdbeek , you're always welcome and bring coworkers! 
@Alanamosse come here!

### CODE REVIEW

**Comment by yvanlebras** [comment]
Let's go for the show!!!

**Comment by dannon** [comment]
@yvanlebras The plan is to leave this as WIP for a while and accumulate contributions here, right?

**Comment by yvanlebras** [comment]
That's exactely what we propose @dannon if you agree that this is a good way to proceed. Tanks for your help!

**Comment by dannon** [comment]
@yvanlebras Sounds good to me!  I just wanted to make sure I labelled the issue correctly :)

File: client/galaxy/scripts/nls/fr/locale.js [hunk 1]
```diff
@@ -246,24 +246,24 @@ define({
     Trackster: false,
     Visualize: false,
     // ---------------------------------------------------------------------------- dataset-error
-    "Any additional comments you can provide regarding what you were doing at the time of the bug.": false,
-    "Your email address": false,
+    "Any additional comments you can provide regarding what you were doing at the time of the bug.": "N'importe quels commentaires que vous pouvez fournir concernant ce que vous faisiez lorsque le bug est apparu",
+    "Your email address": "Votre adresse e-mail",
     Report: false,
-    "Error Report": false,
+    "Error Report": "Rapoprt d'erreur",
```
Comment (FredericBGA) [inline client/galaxy/scripts/nls/fr/locale.js]: une typo : 
Rapport d'erreur

File: client/galaxy/scripts/nls/fr/locale.js [hunk 2]
```diff
@@ -246,24 +246,24 @@ define({
     Trackster: false,
     Visualize: false,
     // ---------------------------------------------------------------------------- dataset-error
-    "Any additional comments you can provide regarding what you were doing at the time of the bug.": false,
-    "Your email address": false,
+    "Any additional comments you can provide regarding what you were doing at the time of the bug.": "N'importe quels commentaires que vous pouvez fournir concernant ce que vous faisiez lorsque le bug est apparu",
```
Comment (FredericBGA) [inline client/galaxy/scripts/nls/fr/locale.js]: Je mettrais au singulier: 
N'importe quel commentaire

File: client/galaxy/scripts/nls/fr/locale.js [hunk 3]
```diff
@@ -272,16 +272,16 @@ define({
...
-    "Create New Folder": false,
+    "Location Details": "Détails de localisation",
+    "Deleting selected items": "Suppression des objets sélectionnés",
+    "Please select folders or files": "S'il vous plaît sélectionnez des dossiers ou fichiers",
+    "Please enter paths to import": "S'il vous plaît entrer des chemins/URLs à importer",
+    "Adding datasets from your history": "Ajouts de jeux de données à partir de votre historique",
+    "Create New Folder": "Créer un nouveau dossier",
     // ---------------------------------------------------------------------------- library-librarytoolbar-view
-    "Create New Library": false,
+    "Create New Library": "Créer une nouvelle librairie",
```
Comment (FredericBGA) [inline client/galaxy/scripts/nls/fr/locale.js]: bibliothèque ?

**Review by FredericBGA — COMMENTED** [review]
Hello

I put some comments.
Nice work.

**Comment by ValentinChCloud** [comment]
@FredericBGA thanks to your comments, the modifications have been done

**Comment by dannon** [comment]
Thanks everyone!  We can open a new one with more changes, as there are any -- this one seems to be done for now.  I also have some ideas about localization file management that I need to write up, which will hopefully make it even easier for us to do this work and generate bundles moving forward.

**Pull request merged**